### PR TITLE
add powerby and presentations pages

### PR DESCRIPTION
**Pull request by helenlly**
1) add powerby page
2) add presentations page

testdemo:  https://testhelendoc.readthedocs.io/en/latest/index.html

### CODE REVIEW

File: readthedocs/source/index.rst [hunk 1]
```diff
@@ -49,6 +49,18 @@ Analytics Zoo includes the **Orca** library that seamlessly scale out your singl
...
+   :maxdepth: 1
+   :caption: Powered By
+   
+   doc/powered-by.md  
+   
+.. toctree::
+   :maxdepth: 1
+   :caption: Presentations
+   
+   doc/presentations.md
```
Comment (jason-dai) [inline readthedocs/source/index.rst:63]: put these two under "Real-World Applications"

**Comment by jason-dai** [comment]
Move the md files to a new `application` folder

**Comment by helenlly** [comment]
done

**Comment by jason-dai** [comment]
LGTM

**Pull request merged**